src/test/java/org/sqlflow/client/SQLFlowRemoteTest.java

```diff
@@ -36,6 +37,22 @@ public void runTest() {
       System.out.println("skip remote test");
       return;
     }
-    assert EnvironmentSpecificSQLFlowClient.hasGoodResponse(client, "SELECT 2");
+    try {
+      client.run("SELECT 2");
+    } catch (Exception e) {
+      System.err.println("encounter an exception while running SQLFlow");
+      assert false;
```

Comment:
We don't need assert false here. Exception will cause the test failure.

Reply:
Good idea.